setup.py

```diff
@@ -34,6 +34,30 @@ def get_path(fname):
     "Markdown>=2.6,<2.7",
 ]
 
+test_requirements = [
+    'django-functest>=1.0,<1.1',
+    'mock>=2.0,<2.1',
+    'pytest>=3.4,<3.5',
+    'pytest-django>=3.1,<3.2',
+    'pytest-cov>=2.4,<2.5',
+    'pytest-pythonpath>=0.7,<0.8',
+]
+
+test_lint_requirements = [
+    'flake8>=3.5,<3.6',
```

Comment:
Agreed! Actually flake8 has been introducing changes that broke linting out of no where... it's best to be explicit about which version of flake8 that linting is supposed to conform to (together with specs in setup.cfg)